src/plans/create/steps/migration-hooks/utils.ts

```diff
@@ -36,3 +40,17 @@ export const validateHookRunnerImage = (value: string): string | true => {
 
   return true;
 };
+
+export const validateHookServiceAccount = (value: string): string | true => {
```

Comment:
Returning a string or a boolean isn't the right combination; it's misleading about what this function does.